### ISSUE TITLE
There is No Linux AppImage Available like OrcaSlicer Official Releases

### ISSUE THREAD

**Issue by timnolte** [FlashForge/Orca-Flashforge#3]
**Is your feature request related to a problem? Please describe.**
The official OrcaSlicer releases provide a Linux AppImage for Linux users, there is no such release available for the FlashForge fork/releases.

**Which printers will be beneficial to this feature**
Adventurer 5M Series

**Describe the solution you'd like**
Unless FlashForge intends to contribute back all Adventurer support work to the official OrcaSlicer releases then there needs to be an Orca-FlashForge release for Linux users as well.

**Describe alternatives you've considered**
There is a FlashPrint release for Linux but there are other capabilities, and even the Adventurer 5M encloser printing instructions recommend using OrcaSlicer for printing.

**Additional context**
This also would support running Orca-FlashForce on Chromebook devices which are common for educational purposes. I will actually be teaching a homeschool course on 3D modeling and printing to students that have only Chromebooks and it would be a huge benefit to allow the use of Orca-FlashForge for these students to slice their 3D models.

**Comment by Iwo24pl** [comment]
Yes, but fortunetly you can build it now for linux directly. Yes, thats some work, but better than nothing.

**Comment by chibiconsulting** [comment]
> Yes, but fortunetly you can build it now for linux directly. Yes, thats some work, but better than nothing.

How?
